README.md

```diff
@@ -1,10 +1,69 @@
+
+* Open Source Advocate: OSPOs can promote the use and/or contribution of open source and best practices across different organizational units. This can help organizations realize the benefits of open source as well as engaging people to contribute to open source projects or start new ones.
+
+* Open Source Environmentalist: OSPOs can help organizations support and sustain open source projects in the long term by addressing issues such as security, maintenance, and project health. This can involve establishing policies and procedures for code review, security vulnerability management, and ongoing maintenance and support through funding and/or contributions. By doing so, OSPOs can help ensure that open source projects remain healthy and continue to benefit the wider community.
+
+**[HOW]** The way the people behind an OSPO achieve this is by creating and maintaining a framework covering the following aspects: strategy, governance, compliance, and community engagement. The OSPO's strategy focuses on aligning the organization's open source goals with its overall organization objectives.
+
+> OSPOs can be formed in various sectors, regions, and organizational sizes including private and public organizations such as academics, NGOs, foundations, governments, and public administrations, as well as small, medium, and large companies. It's also important to note that the term "Program" in OSPOs can sometimes be excluded (Open Source Office).
+
+## Basic Segmentation
+
+There is no broad template for building an open source program that applies across all industries, or even across all companies in a single industry. However we can categorize the functions of an OSPO into different categories, depending on the nature of the organization and its motivators.
+
+* `📈 Business-Oriented`: OSPOs established within for-profit organizations, such as enterprises, 
+that are primarily focused on creating business value through the use of open source software. 
+
+    * **Drivers of Motivation:** Innovation / Risk Management / Legal Compliance / Talent Retention
+
+* `🎓 Educational-Oriented`: OSPOs established by educational institutions, such as universities 
+or schools, that are focused on using open source software to support teaching, research and learning 
+activities.
 
-There is no broad template for building an open source program that applies across all industries, or even across all companies in a single industry. However we can categorize the functions of a typical OSPO into three categories: Legal Risk Mitigation, Improving Engineers’ Practices and Enabling Financial Benefits. (A mnemonic: Fear, Love and Money)
+    * **Drivers of Motivation:** Curriculum development / Student Engagement / Open Data / Knowledge Sharing
+
+* `👩‍🏫 Business-Educational`: OSPOs established within for-profit organizations that provide training and certification and are likely to collaborate
```

Comment:
Maybe instead of adding this mixed form of category, it would be better to add a sentence at the top which says that there can be mixed forms. Business-Eductional is not the only possible mix. Gov- and NonGov-Oriented could for example also be mixed with Educational or have certain business-related motivations.